docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -2,8 +2,135 @@
 sidebar_label: With Argo CD
 ---
 
-# Advanced Installation with Argo CD
+# Installation with Argo CD
 
-:::warning
-Placeholder
+This section outlines a few generalized approaches to installing and managing Kargo with non-default configuration options using Argo CD.
+
+:::note
+This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please refer to [Basic Installation](../../operator-guide/basic-installation#prerequisites) for more details.
+:::
+
+## Direct Argo CD Application 
+
+The most common way to deploy Kargo using Argo CD is to create an `Application` and use the Helm chart directly. Using this method, you can use the `.spec.source.helm.parameters` section to specify any parameters you may need. This is the most straightforward way to deploy Kargo using Argo CD.
+
+:::info
+The parameters used are just examples, and you should use the values that are appropriate for your environment. Detailed information about available options can also be found in the [Kargo Helm Chart's README.md](https://github.com/akuity/kargo/tree/main/charts/kargo).
 :::
+
+```yaml
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kargo
+  namespace: argocd
+spec:
+  project: default
+  destination:
+    namespace: kargo
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: ghcr.io/akuity/kargo-charts
+    chart: kargo
+    targetRevision: 1.2.0
+    helm:
+      parameters:
+        - name: api.adminAccount.passwordHash
+          value: "$$2a$$10$$Zrhhie4vLz5ygtVSaif6o.qN36jgs6vjtMBdM6yrU1FOeiAAMMxOm"
+        - name: controller.logLevel
+          value: "DEBUG"
+        - name: api.adminAccount.tokenTTL
+          value: "24h"
+        - name: api.adminAccount.tokenSigningKey
+          value: "iwishtowashmyirishwristwatch"
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+    - CreateNamespace=true
+```
+
+:::info
+If using the `api.adminAccount.passwordHash` parameter in this method, you must escape the `$` character with `$$` to prevent Helm from interpreting it as a variable. Please see [this discussion](https://discord.com/channels/1138942074998235187/1138946346217394407/1267966083168469102) for more information.
+:::
+
+Conversely, insetad of using the `parameters` field under the `.spec.source.helm` section; you can use the `values` block or `valuesObject` object to specify the values for the Kargo Helm chart. Below is an example of how to use `valuesObject` to specify the values.
```

Comment:
```suggestion
Alternatively, instead of using `.spec.source.helm`'s `parameters`, you can use the either of the `values` `valuesObject` fields to specify configuration options for
the chart:
```